src/main/java/frc/team1816/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,56 @@
+package frc.team1816.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Drivetrain extends Subsystem {
+    private TalonSRX leftMain, leftSlaveOne, leftSlaveTwo, rightMain, rightSlaveOne, rightSlaveTwo;
+
+    private double leftPower, rightPower = 0;
+
+    public Drivetrain(int leftMain, int leftSlaveOne, int leftSlaveTwo, int rightMain, int rightSlaveOne,
+                      int rightSlaveTwo) {
+        super();
+        this.leftMain = new TalonSRX(leftMain);
+        this.leftSlaveOne = new TalonSRX(leftSlaveOne);
+        this.leftSlaveTwo = new TalonSRX(leftSlaveTwo);
+
+        this.rightMain = new TalonSRX(rightMain);
+        this.rightSlaveOne = new TalonSRX(rightSlaveOne);
+        this.rightSlaveTwo = new TalonSRX(rightSlaveTwo);
+
+        this.leftMain.setInverted(true);
+        this.leftSlaveOne.setInverted(true);
+        this.leftSlaveTwo.setInverted(true);
+
+        this.leftMain.setNeutralMode(NeutralMode.Brake);
+        this.leftSlaveOne.setNeutralMode(NeutralMode.Brake);
+        this.leftSlaveTwo.setNeutralMode(NeutralMode.Brake);
+
+        this.rightMain.setNeutralMode(NeutralMode.Brake);
+        this.rightSlaveOne.setNeutralMode(NeutralMode.Brake);
+        this.rightSlaveTwo.setNeutralMode(NeutralMode.Brake);
+
+        this.leftSlaveOne.set(ControlMode.Follower, leftMain);
+        this.leftSlaveTwo.set(ControlMode.Follower, leftMain);
+
+        this.rightSlaveOne.set(ControlMode.Follower, rightMain);
+        this.rightSlaveTwo.set(ControlMode.Follower, rightMain);
+    }
+
+    public void setDrivetrain(double leftPower, double rightPower){
+        this.leftPower = leftPower;
+        this.rightPower = rightPower;
+        update();
+    }
+
+    public void update(){
```

Comment:
This looks like a copy & paste from the old code, and while not technically wrong we could update it to better reflect the new WPILIB.  In the old code we extended Subsystem1816, and this had an update() method that was called repeatedly to update the subsystem.

However now the update() method is not getting called by anything, but there is a new periodic() method in Subsystem that we could override instead.  So if we change this method name to periodic() it should get called automatically.  And we can remove the call to update() from setDrivetrain().